packages/common-lib/src/components/FilterButton.js

```diff
@@ -102,20 +124,29 @@ export default function FilterButton({
                 colorScheme='button'
                 variant='outline'
                 px='5'
+                _text={{
+                  textTransform: 'inherit',
+                  fontWeight: '500',
+                  fontSize: '14px'
+                }}
```

Comment:
Possible to use the same H1, H2 that we created earlier here.

Reply:
that is in another PR